utils/abilities.cfg

```diff
@@ -965,6 +964,40 @@ Enemy units cannot see this unit during dawn or dusk, except if they have units
         [/filter_self]
     [/regenerate]
 #enddef
+#define ABILITY_DEVOURING_EMBERS
+    [regenerate]
+        value=16
+        id=devouring embers
+        name=_ "devouring embers"
+        description=_ "This unit heals 16 HP per turn when above lava. If poisoned, the poison will be cured instead."
+        name_inactive=_ "devouring embers"
+        special_note = _" This unit regenerates, which allows it to heal as though always stationed in a village, if it is above lava."
+        affect_self=yes
+        [filter_self]
+            [filter_location]
+                terrain=Ql*,Mv*
+            [/filter_location]
+        [/filter_self]
+        poison=cured
+    [/regenerate]
+#enddef
+#define ABILITY_PROPEL
+    [dummy]
+        id=propel
+        name= _ "propel"
+        description= _ "When an ally is standing next to this unit at the beginning of its turn, it gains two movement points."
```

Comment:
There is an identical ability named `wardrums`. Please change it to 3 movement points or add some other difference. Or reuse `wardrums`.

Reply:
I knew I'd seen that somewhere.  Done.